### PR TITLE
Prepare for new release of snowflake which does not have connect_params

### PR DESCRIPTION
**Pull request by ahuang11**
## Summary

The following PR removes self.connect_params and migrates keywords/method to SnowflakeConnector. Wondering if we should rename `credentials` to `block`?
https://github.com/PrefectHQ/prefect-snowflake/pull/24

Do not merge until that PR has been merged

## Checklist
- [ ] Summarized PR's changes in [CHANGELOG.md](https://github.com/PrefectHQ/prefect-dbt/blob/main/CHANGELOG.md)

### CODE REVIEW

File: tests/cli/configs/test_snowflake.py [hunk 1]
```diff
@@ -33,21 +40,3 @@ def test_snowflake_target_configs_get_configs(schema_in_target_configs):
         )
         expected_v = expected[k]
         assert actual_v == expected_v
-
-
-def test_snowflake_target_configs_get_configs_missing_schema():
```
Comment (ahuang11) [inline tests/cli/configs/test_snowflake.py:38]: Removed this because schema is no longer optional in prefect-snowflake

**Comment by ahuang11** [comment]
image

**Review by desertaxle — CHANGES_REQUESTED** [review]
Can you pin `prefect-snowflake` to `>= 0.2.0`?

**Comment by ahuang11** [comment]
I think I did already in setup.py

**Review by desertaxle — APPROVED** [review]

**Pull request merged**